sdk-config.json

```diff
@@ -1,21 +1,15 @@
 {
   "comment_sdk_config": "This is the configuration file for the Angular-SDK",
   "authConfig": {
-    "comment_authConfig": "authConfig is the block for authentication config entries",
-
-    "accessAuthUris_comment": "Full paths to Infinity server access token and authorization APIs",
-    "accessTokenUri": "https://localhost:1080/prweb/PRRestService/oauth2/v1/token",
-    "authorizationUri": "https://localhost:1080/prweb/PRRestService/oauth2/v1/authorize",
-    "authorityUri": "https://localhost:1080/prweb/PRRestService/oauth2/v1",
-    
-    "authQueryParams_comment": "Additional parameters that are passed to the OpenID Connect (OIDC) Manager",
-    "authQueryParams": { "authentication_service": "pega" },
+    "authConfig_comment": "Optional Full paths to Infinity server OAuth endpoints (Only needed if using custom OAuth service)",
+    "authConfig_comment2": "Optional attributes are: authorize, token, revoke, authService, silentTimeout",
+    "authService": "pega",
         
     "mashupClient_comment": "Client ID and Client secret from the OAuth 2.0 Client Registration record used for mashup use case",
     "mashupClient_comment2": "See SDK Guide for instructions on how to generate and obtain the proper values for the following entries",
-    "mashupClientId": "12595401538569726922",
-    "mashupClientSecret": "",
-    "mashupGrantType": "client_credentials",
+    "mashupClientId": "88249793673398954156",
+    "mashupUserIdentifier": "customer@mediaco",
+    "mashupPassword": "cGVnYQ==",
```

Comment:
Please remove this since password value since previous merges of client secret have resulted in problems with SecOps. We'll need to document how to enter the correct value.